manifests/private.yaml

```diff
@@ -49,6 +49,9 @@ spec:
     - mountPath: /etc/my.cnf.d/max-allowed-packet.cnf
       name: mariadb-config-max-allowed-packet
       readOnly: True
+    - mountPath: /etc/my.cnf.d/innodb-log-file-size.cnf
+      name: mariadb-config-innodb-log-file-size
+      readOnly: True
```

Comment:
I would *love* to see us stop mounting stuff into the containers ;) Given this is static, can we just bake it into the image?

And, if we do add a env var - I'd still prefer to see it in the image, with the entrypoint rendering the file based on the env var / default value.

Reply:
I agree this is not the best way, and that we should stop doing this, but I find more concerning having some settings in a file and some others being rendered from an envvar, and this is why I went with this option. It's as static as everything under https://github.com/kubic-project/caasp-container-manifests/tree/master/config/mysql.

Summing up: I have no objections in adding this to the image, but I think if we do that we should also move the settings under https://github.com/kubic-project/caasp-container-manifests/tree/master/config/mysql to basically do the same thing; so at least we are consistent.

Reply:
Makes sense. LGTM then. We really need to clean it all up at some point though :)

Reply:
The change is a bit messy, this now depends on https://github.com/kubic-project/container-images/pull/31 for 4.0, and the backport needs a SR created against IBS, so I'll wait until https://github.com/kubic-project/container-images/pull/31 gets reviewed to continue with the SR.